.github/workflows/push.yml

```
@@ -10,6 +10,7 @@ jobs:
     env:
       ROCKSDB_LIB_DIR: /usr/lib/
       RUST_BACKTRACE: 1
+      CLIPPY_LINTS: '-D warnings -D clippy::cast-lossless -D clippy::cast-possible-truncation -D clippy::cast-possible-wrap -D clippy::cast-precision-loss -D clippy::cast-sign-loss -D clippy::checked-conversions'
```

Hopefully this syntax works

```suggestion
      CLIPPY_LINTS: >-
        -D warnings
        -D clippy::cast-lossless
        -D clippy::cast-possible-truncation
        -D clippy::cast-possible-wrap
        -D clippy::cast-precision-loss
        -D clippy::cast-sign-loss
        -D clippy::checked-conversions
```